src/main_kmeans.R

```diff
@@ -0,0 +1,7 @@
+source("src/grouping_experiment.R")
+source("src/kmeans_wrapper.R")
+
+message("START")
+grouping_experiment(grouping_kmeans())
```

Comment:
passing here extract_normal_anomaly fun handler looks not bad and solution seems to look a bit more general

Reply:
Done